src/module-vsbridge-indexer-catalog/Plugin/Controller/Catalog/Adminhtml/Product/Action/AttributeSavePlugin.php

```diff
@@ -0,0 +1,85 @@
+<?php declare(strict_types=1);
+/**
+ * @package Divante\VsbridgeIndexerCatalog
+ * @author Marek Mularczyk <mmularczyk@divante.pl>
+ * @copyright 2019 Divante Sp. z o.o.
+ * @license See LICENSE_DIVANTE.txt for license details.
+ */
+
+namespace Vsf\Indexer\Plugin\Controller\Catalog\Adminhtml\Product\Action;
+
+use Magento\Backend\Model\View\Result\Redirect;
+use Magento\Catalog\Controller\Adminhtml\Product\Action\Attribute\Save;
+use Magento\Catalog\Helper\Product\Edit\Action\Attribute;
+use Magento\Framework\Indexer\IndexerInterface;
+use Magento\Indexer\Model\Indexer\CollectionFactory as IndexerCollectionFactory;
+
+/**
+ * Class AttributeSavePlugin
+ */
+class AttributeSavePlugin
+{
+    /**
+     * @var Attribute
+     */
+    private $attributeHelper;
+
+    /**
+     * @var IndexerCollectionFactory
+     */
+    private $indexerColFactory;
+
+    /**
+     * AttributeSavePlugin constructor.
+     *
+     * @param Attribute $attributeHelper
+     * @param IndexerCollectionFactory $indexerColFactory
+     */
+    public function __construct(Attribute $attributeHelper, IndexerCollectionFactory $indexerColFactory)
+    {
+        $this->attributeHelper = $attributeHelper;
+        $this->indexerColFactory = $indexerColFactory;
+    }
+
+    /**
+     * @param Save $subject
+     * @param Redirect $result
+     *
+     * @return Redirect
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function afterExecute(Save $subject, $result)
+    {
+        $productIds = $this->attributeHelper->getProductIds();
+        $indexers = $this->getIndexers();
+
+        /** @var IndexerInterface $indexer */
+        foreach ($indexers as $indexer) {
+            $indexer->reindexList($productIds);
+        }
+
+        return $result;
+    }
+
+    /**
+     * @see \Divante\VsbridgeIndexerCore\Console\Command\Console\Command\RebuildEsIndexCommand
+     *
+     * @return IndexerInterface[]
+     */
+    protected function getIndexers(): array
+    {
+        /** @var IndexerInterface[] */
+        $indexers = $this->indexerColFactory->create()->getItems();
+        $vsbridgeIndexers = [];
+
+        /** @var IndexerInterface $indexer */
+        foreach ($indexers as $indexer) {
+            if (!$indexer->isScheduled() && substr($indexer->getId(), 0, 9) === 'vsbridge_') {
+                $vsbridgeIndexers[] = $indexer;
+            }
+        }
```

Comment:
we don't want to run all indices... we can have a lot of them, we have indexer for cms block, pages, reviews - this one shouldn't be executed. (+ users can add also other custom indices)

After  changing product attributes, we should only run reindex for `vsbridge_product_indexer` indexer.

so please use 
`Divante\VsbridgeIndexerCatalog\Model\Indexer\ProductProcessor`
 instead.
Take a look at plugin:
https://github.com/DivanteLtd/magento2-vsbridge-indexer/blob/develop/src/module-vsbridge-indexer-catalog/Plugin/Indexer/Product/Save/UpdateProductDataPlugin.php

Reply:
fixed